app/config/admin/security.yml

```diff
@@ -23,3 +23,27 @@ security:
     dev:
       pattern: ^/(_(profiler|wdt)|css|images|js)/
       security: false
+
+    main:
+      lazy: true
+      provider: admin
+      form_login:
+        login_path: admin_login
+        check_path: admin_login
+        username_parameter: email
+        password_parameter: passwd
+        success_handler: PrestaShopBundle\Security\Admin\AdminAuthenticationSuccessHandler
+      remember_me:
+        secret: "%kernel.secret%"
+        lifetime: 3600
+        remember_me_parameter: stay_logged_in
+        signature_properties: [ 'password' ]
+      logout:
+        path: /logout
+        target: admin_login
+
+  access_control:
+    - { path: ^/login, roles: PUBLIC_ACCESS }
+    - { path: ^/request-password-reset, roles: PUBLIC_ACCESS }
+    - { path: ^/reset-password, roles: PUBLIC_ACCESS }
+    - { path: ^/, roles: IS_AUTHENTICATED }
```

Comment:
@jolelievre Please make sure that this works with search cron, wouldn't it be blocked?

Reply:
I didn't test it yet but it will likely fail, this has already been anticipated in the follow up tasks of this epic https://github.com/PrestaShop/PrestaShop/issues/36001

Reply:
It is possible to redo the Search controller under Symfony, and give it public access via attribute isGranted. For later

Reply:
Yep but that's the easy way I guess 😅 we still have legacy controllers in modules that will need this feature